### DISCUSSION TITLE
Capacitative sensors of esp32

### DISCUSSION THREAD

**Discussion by vali20** [bxparks/AceButton#116] [Q&A]
Is there any chance to work with the capacitive sensors of the ESP32?
Thanks!

**Comment by bxparks** [comment]
Maybe? I don't know anything about the capacitive sensos that seem to be built into some pins on the ESP32 and some other microcontrollers. No idea how the hardware works, no idea how the driver software works.

One important feature of the AceButton library is to perform debouncing, which is a problem that mechanical switches are prone to have. I have no idea if debouncing is needed for capacitive sensors.

If you figure this stuff out, and find that integration with AceButton is useful, I'd appreciate a note here, or even better, a pull request.
